.github/workflows/ci.yml

```diff
@@ -8,13 +8,14 @@ on:
       - "snapshot-*"
   pull_request:
 
+permissions:
+  contents: write
```

Comment:
`contents` was used for rebar3 releases. For this one you need it to be `issues` to create comments.